ED/protocol.html

```diff
@@ -934,22 +934,49 @@ <h2 property="schema:name">Notifications</h2>
+                        [<cite><a class="bibref" href="#bib-solid-notifications-protocol">SOLID-NOTIFICATIONS-PROTOCOL</a></cite>]
+                      </span>
+                    </span>
+                  </p>
+
+                  <p>
+                    <span about="" id="client-notification-api" rel="spec:requirement" resource="#client-notification-api">
+                      <span property="spec:statement">
+                        <span rel="spec:requirementSubject" resource="spec:Client">Clients</span>
+                        <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> implement the
+                        <cite><a href="https://solidproject.org/TR/notifications-protocol">Solid Notifications Protocol</a></cite>
+                        [<cite><a class="bibref" href="#bib-solid-notifications-protocol">SOLID-NOTIFICATIONS-PROTOCOL</a></cite>]
+                      </span>
+                    </span>
+                  </p>
 
                   <p><em>The following is non-normative.</em></p>
 
-                  <p>The <cite>Solid WebSockets API</cite> (Unofficial Draft) has been the common protocol for many years. That draft does not include an authentication mechanism, and therefore the Protocol will transition to a new design. The new design is currently at <cite><a href="https://solid.github.io/notifications/protocol" rel="cito:citesAsPotentialSolution">Solid Notifications Protocol</a></cite> [<cite><a class="bibref" href="#bib-solid-notifications-protocol">SOLID-NOTIFICATIONS-PROTOCOL</a></cite>]. It is planned to include both security through authentication, and also common formats with other forms of real-time notification in the Solid ecosystem.</p>
+                  <p>
+                    The <cite>Solid WebSockets API</cite> (Unofficial Draft)
+                    [<cite><a class="bibref" href="#bib-solid-websockets-api">SOLID-WEBSOCKETS-API</a></cite>]
+                    has been the common notification protocol for many years. That draft does not include an authentication mechanism,
+                    and therefore this Protocol has transitioned to require the <a href="https://solidproject.org/TR/notifications-protocol">Solid Notifications Protocol</a>.
+                  </p>
 
-                  <p>Both client and server implementations should provide the existing protocol, and should transition to providing both protocols as the new one becomes available..</p>
-
-                  <p>The future directions of the protocol include moving from a simple one-bit notification that a resource has changed, requiring the client to reload the resource, to adding <code>PATCH</code> information in the notification so the client can calculate the new state immediately.</p>
+                  <p>
+                    Existing client and server implementations should begin providing support for the new notification protocol while supporting backwards compatability, as appropriate.
```

Comment:
```suggestion
                    Existing client and server implementations should begin providing support for the new notification protocol while supporting backwards compatibility, as appropriate.
```

But I find the statement hard to interpret; "as appropriate". "As they deem appropriate" perhaps?